packages/manager/tools/static-analysis-kit/src/adapters/eslint/rules/eslint-react.ts

```diff
@@ -43,6 +43,9 @@ export const reactEslintConfig: Linter.FlatConfig = {
     ...reactPlugin.configs.recommended.rules,
     ...reactHooksPlugin.configs.recommended.rules,
 
+    // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md#when-not-to-use-it
+    'react/react-in-jsx-scope': 'off', // Disable React import rule for JSX
```

Comment:
It could be great to extend from `'plugin:react/jsx-runtime'` , because it turns off 2 rules, as explained in the official documentation:
https://github.com/jsx-eslint/eslint-plugin-react?tab=readme-ov-file#configuration-legacy-eslintrc-

https://github.com/jsx-eslint/eslint-plugin-react/blob/c8917b0885094b5e4cc2a6f613f7fb6f16fe932e/index.js#L163-L176

I think it's much cleaner than turning-off rules manually, what do you think.